### PR TITLE
Version 0.1.3

### PR DESCRIPTION
**Pull request by agrawalsuneet09**
## What?

Changed the constructor and passed the username, password and projectId separately
## Why

Google was showing the warning while passing the url directly

### CODE REVIEW

**Comment by dushyantsingh23** [comment]
Why am I seeing this again???

**Comment by rachitmishra** [comment]
@dushyantsingh23 public repo :-)

File: .gradle/2.2.1/taskArtifacts/cache.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+#Tue Jun 28 15:49:24 IST 2016
```
Comment (rachitmishra) [inline .gradle/2.2.1/taskArtifacts/cache.properties]: Add .gitignore

File: src/com/joshdholtz/sentry/Sentry.java [hunk 2]
```diff
@@ -82,7 +78,10 @@
 	private String android_phone_product;
 	private String app_version_code;
 	private String app_version_name;
-	
+
+	private String publicKey, secretKey;
```
Comment (rachitmishra) [inline src/com/joshdholtz/sentry/Sentry.java]: mPublicKey

**Comment by rachitmishra** [comment]
Review Ok.

**Pull request merged**